cmd/openshift-install/BUILD.bazel

```diff
@@ -19,6 +20,6 @@ go_binary(
     embed = [":go_default_library"],
     # Use pure to build a pure-go binary.
     # This has the nice side effect of making the binary statically linked.
-    pure = "on",
+    pure = "off",
```

Comment:
Do you know why we need this?  It may break @steveeJ (#243, #300).

Reply:
`github.com/libvirt/libvirt-go` requires `cgo`

Reply:
> I can drop it for now.

I'd favor that :-)